Sources/InternxtSwiftCore/Services/Network/Encrypt.swift

```diff
@@ -74,6 +73,8 @@ public struct Encrypt {
             sha256Hash.append(contentsOf: bytes)
         }
         
+        stream.close()
```

Comment:
Should the hasher be closed, also?

Reply:
The hasher is "done" once we call `.finalize()` if you attempt to update it after that, it throws an error, also note the hasher is not an stream